### ISSUE TITLE
Improve Storage Drawers integration

### ISSUE THREAD

**Issue by Nagshell** [CyclopsMC/IntegratedDynamics#351]
Hello, I tried to setup some simple setups, but I discovered that when I attached inventory reader onto Storage Drawer, it's "Fill ratio" was only showing one of three values - 0, 0.5 and 1.
And 0.5 was shown all the time whenever Drawer wasn't empty nor full. 

I wanted to use fill ratio to accommodate for future upgrades that would make simple item count not efficient.

I'm attaching screenshot showing That there is only 7 out of 2048 items in drawer and it's showing fill ratio of half.

![image](https://user-images.githubusercontent.com/17276063/32501794-42fc167e-c3d9-11e7-854b-9f4e7129b316.png)

**Comment by Nagshell** [comment]
I just noticed that Fill ratio is working a little bit differently than I thought it was.

Still there is no way of detecting how far to full drawer, because variables behave in the same way on not upgraded and upgraded drawers.
I guess I'll use item count for now.
Not sure if it's possible to do something with this issue then, but maybe some additional compatibility in the future would be possible.

**Comment by rubensworks** [comment]
Indeed, the fact that Storage Drawers basically has stacksizes > 64, this introduces some issues. In the future, we should probably add custom support for the SD API.
